src/OrganisationUnitTree/Content.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+
+import { useChildReloadIds } from './useChildReloadIds'
+
+export const Content = ({
+    open,
+    props,
+    error,
+    loading,
+    children,
+    OrganisationUnitNode,
+    childrenIds,
+    orgUnitsPathsToInclude,
+    idsThatShouldBeReloaded,
+}) => {
+    const childIds = useChildReloadIds(childrenIds, idsThatShouldBeReloaded)
+    const showChildren = !loading && !error && open
+
+    if (!showChildren) {
+        // @TODO Ask Joe Cooper
```

Comment:
@cooper-joe I need your help here. Need to know how to style the loading state.
My current thought is to display the cirular spinner where the arrow icon is, dunno if that's what you'd like as well

Reply:
@cooper-joe could you comment on this?

Reply:
Ideally we would show the loading spinner and text as a child of the parent that triggered the loading:
![image](https://user-images.githubusercontent.com/33054985/72797298-25075600-3c41-11ea-8881-cfd93a5ad0c7.png)
(spinner is 18px, text is 13px grey700)

Clicking the arrow again to close the parent node would cancel the loading.

Reply:
Cheers, I like it